test_voc.ino

```diff
@@ -1,14 +1,17 @@
-#if false
+#if true
```

Comment:
I don't feel strongly about this, but do we want that `#if true` to be on master?